File: JAVA/Program/Tv.java
import java.util.ArrayList;
public class Tv extends Produk {
    private int UkuranLayar;
    private String resolusi;
    private String tipeLayar;

    public Tv(){
        
    }
    //constructor
    public Tv(int id_produk, String merk, int harga, int stok, int UkuranLayar, String resolusi, String tipeLayar) {
        super(id_produk, merk, harga, stok);
        this.UkuranLayar = UkuranLayar;
        this.resolusi = resolusi;
        this.tipeLayar = tipeLayar;
    }

    // gtter dan setter
    public int getUkuranLayar() {
        return UkuranLayar;
    }

    public void setUkuranLayar(int UkuranLayar) {
        this.UkuranLayar = UkuranLayar;
    }

    public String getResolusi() {
        return resolusi;
    }

    public void setResolusi(String resolusi) {
        this.resolusi = resolusi;
    }

    public String getTipe() {
        return tipeLayar;
    }

    public void setTipe(String tipeLayar) {
        this.tipeLayar = tipeLayar;
    }
}

File: PHP/Program/Tv.php
<?php
require_once "Produk.php";

class Tv extends Produk
{
    private int $UkuranLayar;
    private string $resolusi;
    private string $tipeLayar;

    // Konstruktor default
    public function __construct(int $id_produk = 0,string $merk = "",int $harga = 0,int $stok = 0, string $gambar = "",int $UkuranLayar = 0,string $resolusi = "", string $tipeLayar = "" ) {
        // Panggil konstruktor parent
        parent::__construct($id_produk, $merk, $harga, $stok, $gambar);
        $this->UkuranLayar = $UkuranLayar;
        $this->resolusi = $resolusi;
        $this->tipeLayar = $tipeLayar;
    }

    // Getter dan Setter
    public function getUkuranLayar(): int
    {
        return $this->UkuranLayar;
    }

    public function setUkuranLayar(int $UkuranLayar)
    {
        $this->UkuranLayar = $UkuranLayar;
    }

    public function getResolusi(): string
    {
        return $this->resolusi;
    }

    public function setResolusi(string $resolusi)
    {
        $this->resolusi = $resolusi;
    }

    public function getTipe(): string
    {
        return $this->tipeLayar;
    }

    public function setTipe(string $tipeLayar)
    {
        $this->tipeLayar = $tipeLayar;
    }
}
?>
